src/app/modules/spcp/spcp.controller.ts

```diff
@@ -32,20 +32,30 @@ export const handleRedirect: RequestHandler<
   { authType: AuthType.SP | AuthType.CP; target: string; esrvcId: string }
 > = (req, res) => {
   const { target, authType, esrvcId } = req.query
+  const logMeta = {
+    action: 'handleRedirect',
+    ...createReqMeta(req),
+    authType,
+    target,
+    esrvcId,
+  }
+  if (authType !== AuthType.SP && authType !== AuthType.CP) {
```

Comment:
Can we not validate this with Joi?

Reply:
lolol there is already joi validation but I added these additional checks because of this comment you left in the other PR: https://github.com/opengovsg/FormSG/pull/635#discussion_r523854401

Reply:
if you feel that Joi is sufficient then I'd also rather remove these checks

Reply:
aiyo i didn't know already had joi validation haha didn't connect; if got joi validation then don't need here liow, since literally only SP/CP can pass through

Reply:
eh don't liddat lol there's nothing to forgive

Reply:
but ok I shall remove the checks